src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -229,6 +229,34 @@ class E
             comp.VerifyDiagnostics();
         }
 
+        [Fact, WorkItem(49628, "https://github.com/dotnet/roslyn/issues/49628")]
+        public void AmbigCtor()
```

Comment:
>AmbigCtor [](start = 20, length = 9)

We should also test a case when an ambiguous (against primary constructor) copy constructor is declared explicitly.

Reply:
Note, prior to this change we already reported an expected error for a scenario like that:
```
record R(R y)
{
    public R(R x) {}
}
``` 

error CS0111: Type 'R' already defines a member called '.ctor' with the same parameter types

There is no need to change error reporting for a scenario like that.

---
In reply to: [543414763](https://github.com/dotnet/roslyn/pull/49975#discussion_r543414763) [](ancestors = 543414763)